packages/shopify-codemod/transforms/assert/call-count-equals-to-assert-called.js

```diff
@@ -0,0 +1,49 @@
+import {isAssert, isMemberCall} from './utils';
+
+export default function callCountEqualsToAssertCalled({source}, {jscodeshift: j}, {printOptions = {}}) {
+  function intToCallMethodName(callCount) {
+    return {
+      0: 'notCalled',
+      1: 'calledOnce',
+      2: 'calledTwice',
+      3: 'calledThrice',
+    }[callCount];
+  }
+
+  const sourceWithSmallCallCountsRemoved = j(source)
+    .find(j.CallExpression, {
+      callee: isAssert('equal', 'strictEqual', 'deepEqual'),
+      arguments: {
+        0: isMemberCall('callCount'),
+        1: (node) => node.value >= 0 && node.value <= 3,
+      },
+    })
+    .replaceWith(({node: {arguments: [actualArg, expectedArg]}}) => {
+      const assertionName = intToCallMethodName(expectedArg.value);
+
+      return j.callExpression(
+        j.memberExpression(j.identifier('assert'), j.identifier(assertionName), false),
+        [actualArg.object]
+      );
+    })
+    .toSource(printOptions);
```

Comment:
Do we have to do the `toSource` call? I think we can just store `j(source)` and then do two separate `.find().replaceWith()` calls on it, before finally doing the `toSource` at the end.